exercises/e001/results/Frydan/b32gen.py

```diff
@@ -0,0 +1,28 @@
+#!/bin/python3
+
+import argparse
+import random
+
+parser = argparse.ArgumentParser(description='Base32 (RFC 4648) random character string generator')
+meg = parser.add_mutually_exclusive_group()
+parser.add_argument('-c', '--characters', type=int, default=32, metavar='', help='Number of characters to generate (default: 32)')
+meg.add_argument('-l', '--lowercase', action='store_false', help='Output all lowercase (default choice)')
+meg.add_argument('-u', '--uppercase', action='store_true', help='Output all uppercase')
+
+args = parser.parse_args()
+
+def getString(numOfCharacters, notLowercase):	# main function for generating base32 strings
+	buff = ''
+	for i in range(numOfCharacters):
```

Comment:
```suggestion
	for _ in range(numOfCharacters):
```

am I seeing this right, that `i` is unused?

Reply:
Yes You are right, i left it out accidentally after testing the whole range of possible characters.